lynx/common/scene.py

```diff
@@ -40,7 +42,24 @@ def get_object_by_id(self, id: int) -> Optional[Object]:
     def get_objects_by_position(self, position: Vector) -> Optional[List[Object]]:
         return self.get_square(position).objects
 
+    def generate_id(self) -> int:
+        max_id_number = 1000000
```

Comment:
`MAX_ID_NUMBER = 1000000`, because it is a const

Reply:
Why the number is 1000000?
I also recommend creating a static class with common variables.